File: dayTwo/javascript/level two/taskThree.js
const prompt = require("prompt-sync")();

let userNumber = Number(prompt("Enter a Score: "))

if(userNumber >= 50 && userNumber <= 100){
    console.log(`${userNumber} Student passed!`)
    }
else if (userNumber >= 0 && userNumber < 50){
    console.log(`${userNumber} Student Failed!`)
}
else{
    console.log("Invalid Grade Score!")
}

File: dayTwo/python/level two/task_three.py
user_number = int(input("Enter a Score: "))

if(user_number >= 50 and user_number <= 100):
    print(f"{user_number} Student passed!")
elif(user_number >= 0 and user_number < 50):
    print(f"{user_number} Student Failed!")
else:
    print("Invalid Grade Score!")

